### PR TITLE
UI rewrite

### PR DESCRIPTION
**Pull request by ajm13**

### CODE REVIEW

**Review by mjrosengrant — APPROVED** [review]
Overall, very nice job! The plugin is easy to use, the code is well organized, and the annotations make it a joy to read.

I won't stop you from merging over this, but you should really get in the habit of adding unittests for new plugins. Its a hassle, but if you do it right, it adds reliability, and allows you to do development work quickly, without even running the VR app.

File: plugin/MSMSInstance.py [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+    enums.ColorScheme.Monochrome,
+    enums.ColorScheme.Chain,
+    enums.ColorScheme.Hydrophobicity,
+]
+
+if sys.platform == 'linux':
+    AO_PATH = os.path.join(BASE_DIR, 'bin/linux/AOEmbree')
+    MSMS_PATH = os.path.join(BASE_DIR, 'bin/linux/msms')
+    os.environ['LD_LIBRARY_PATH'] = os.path.join(BASE_DIR, 'bin/linux')
+elif sys.platform == 'darwin':
```
[the remark this answers is not in the document]
Comment (ajm13) [reply plugin/MSMSInstance.py:49]: I have tested on Mac (darwin) but not on Windows. I think Xavier had run it on Windows before but I'll test.

File: plugin/MSMSMenu.py [hunk 2]
```diff
@@ -0,0 +1,572 @@
...
+
+    def toggle_surface(self, btn: ui.Button):
+        btn.surface.toggle_visible()
+        self.update_surface_list()
+
+    def delete_surface(self, btn: ui.Button):
+        btn.surface.destroy()
+        self.surfaces.remove(btn.surface)
+        self.update_surface_list()
+        gc.collect()
```
[the remark this answers is not in the document]
Comment (ajm13) [reply plugin/MSMSMenu.py:459]: The saved data for the surfaces can be quite large (sometimes potentially ~1GB), and when deleting surfaces from the list I noticed the memory wouldn't get freed for quite some time. I added the manual gc to speed this up.

[the remark this answers is not in the document]
Comment (mjrosengrant) [reply plugin/MSMSMenu.py:459]: neato! I had no idea that was possible

**Pull request merged**